sdks/python/apache_beam/transforms/dataflow_distribution_counter_test.py

```diff
@@ -71,7 +70,7 @@ def test_translate_to_histogram_with_input_0(self):
 
   def test_translate_to_histogram_with_max_input(self):
     counter = DataflowDistributionCounter()
-    counter.add_input(sys.maxint)
```

Comment:
FYI, sys.maxint isn't in Python 3 so we would have had to change this line sooner or later.